packages/ihe-gateway/server/CodeTemplates/Common Library/Config/Config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This module provides a standardized way to access environment variables.
+ */
+
+function getEnvVar(name) {
+  const value = java.lang.String(Packages.java.lang.System.getenv(name));
+  if (value == null) {
+    throw new Error(`Environment variable ${name} is not defined.`);
+  }
+  return value;
+}
+
+const Config = {
+  getSamlHomeCommunityId: function () {
+    return getEnvVar("SAML_HOME_COMMUNITY_ID");
+  },
+
+  getInboundXcpdUrl: function () {
+    return getEnvVar("INBOUND_XCPD_URL");
+  },
+
+  getInboundDqUrl: function () {
```

Comment:
Should we keep acronyms in all caps? [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#16cd72caa4944a10abfac5bf2d0a13e4)

Reply:
I know, I saw this and I even typed out the names like that at first, but it looked so ugly.. `getInboundDQURL` or even worse `getInboundXCPDURL`..

Reply:
I see what you mean, and agree it can get ugly. I even considered proposing underscore `getInboundDQ_URL`, but that's even worse - especially WRT what's the logic there.

This made me rethink the pattern and look around, I think we should avoid all caps in all places TBH, keeping your recommendation. It's only odd b/c we're used to see acronyms in all caps when writing english, but if we need to smash them together it becomes super bad to read.

I'll bring this up on Slack.

Reply:
Started writing about this on [the Notion page](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#16cd72caa4944a10abfac5bf2d0a13e4) (see comments).